doc/wiki/notes/ch4_inline.md

```diff
@@ -1,83 +1,86 @@
    functions (both declarations `netmod.h` and definitions). Move
-   wrapper funcs in `netmod_impl.h` to `netmod/src/netmod_impl.c`.
+   wrapper functions in `netmod_impl.h` to `netmod/src/netmod_impl.c`.
 4. Make internal funcs “private”.
-5. Remove the *.h file if they are empty.
-6. Keep CVAR to the place where it is used.
-
-For netmod, why do we need to create the #define in `*_noinline.h`?
-
-  The problem is sort of backward from the case when NETMOD_INLINE
-  disabled---both netmod_impl.c/h and the ofi_init.h/c will have their
-  own MPIDI_NM_* (the one in netmod_impl.c/h is the wrapper). To solve
-  this name collision, we have to change the name of the uninlined
-  functions to MPIDI_OFI_*. Then we run into the problem when
-  NETMOD_INLINE is enabled, the function name does not match the defined
-  interface name anymore, hence the #define solution.
-
-Why the MPIDI_NM_comm_get_gpid function is moved to ofi_proc.h?
-
-  This function is used in MPIDIG RMA code and need to be inlined.
-  Because it was the only function in ofi_init.h that should be inlined
-  and it is ugly to keep ofi_init.h just for this function.
-
-  It cannot be put in ofi_impl.h because ofi_impl.h is included in many
-  .c file which in turn will create their own copy of
-  MPIDI_NM_comm_get_gpid and conflict with each other. Therefore, I put
-  it in ofi_proc.h.
-
-Tips for resolving conflict when rebasing.
-
-  If you run into conflict in rebasing the file that is uninlined, try
-  doing a diff between the original .h file and the new .c file. I tried
-  to maintain the relative location of the functions when moving them to
-  the .c file so that it is easier to do a diff in editor. I would
-  usually do something like ```vimdiff ofi_win.h ofi_win.c``` to figure
-  out what was changed during conflict resolving.
+5. Remove the `*.h` file if it is empty.
+6. Keep `CVAR` where it is used.
+
+## For netmod, why do we need to create the `#define` in `*_noinline.h`?
+
+The problem is sort of backwards from the case when `NETMOD_INLINE` is
+disabled. Both `netmod_impl.c/h` and the `ofi_init.h/c` will have their own
+`MPIDI_NM_*` (the one in `netmod_impl.c/h` is the wrapper). To solve this name
+collision, we have to change the name of the uninlined functions to
+`MPIDI_OFI_*`. Then we run into the problem when `NETMOD_INLINE` is enabled.
+The function name does not match the defined interface name anymore, hence the
+`#define` solution.
+
+## Why is `MPIDI_NM_comm_get_gpid` moved to `ofi_proc.h`?
+
+This function is used in `MPIDIG RMA` code and needs to be inlined. Because it
+was the only function in `ofi_init.h` that should be inlined and was ugly to
+keep `ofi_init.h` just for this function, it was moved.
+
+It cannot be put in `ofi_impl.h` because `ofi_impl.h` is included in many
+`.c` files which in turn will create their own copy of `MPIDI_NM_comm_get_gpid`
+and conflict with each other. Therefore, it was put in `ofi_proc.h`.
+
```

Comment:
The above two paragraphs explain a minor technical rationale or workaround. It is not critical for understanding and maintenance. Suggest removal.

Reply:
Removed.